scheduler.rst

```diff
@@ -0,0 +1,76 @@
+Scheduler
+=========
+
+The Symfony Scheduler is a powerful and flexible component designed to manage the scheduling of tasks within your PHP application.
+
+Introduction to the case:
+-------------------------
+
+- Provide a case scenario of sending tasks as messages to different handlers.
+- Reference Symfony Messenger for the non recurring approach :doc:`Messenger component docs </components/messenger>`.
+- To emphasize that this doesn't cover every situation and that certain cases will require recurrence, we can provide a few examples.
+
+On the basis of a case study in the context of a full stack Symfony application, we will cover the following topics:
+
+Symfony Scheduler basics:
+-------------------------
+
+Exploring differences and parallels between Symfony Messenger and Symfony Scheduler.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Outline the key differences and similarities between Symfony Messenger and Symfony Scheduler.
+- Compare the different cycles.
+- Introduce the Scheduler Transport and its particularity: generate directly the different messages to handle
+
+The Concept of Recurring Message in Symfony Scheduler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A message associated with a Trigger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- The importance of setting the frequency for your messages: Introduce the concept of a Trigger in Symfony Scheduler encompassing all the different types.
```

Comment:
When covering custom triggers, an interesting example could be a trigger wrapper that excludes holidays. Cron task that runs every day at midnight, except "holidays" (defined by some kind of lookup table).